plexapi/base.py

```diff
@@ -439,7 +439,7 @@ def _getAttrValue(self, elem, attrstr, results=None):
         attrstr = parts[1] if len(parts) == 2 else None
         if attrstr:
             results = [] if results is None else results
-            for child in [c for c in elem if c.tag.lower() == attr.lower()]:
+            for child in filter(lambda c: c.tag.lower() == attr.lower(), elem):
```

Comment:
What about a generator expression? I think it's more readable than `filter()`.
```suggestion
            for child in (c for c in elem if c.tag.lower() == attr.lower()):
```